src/extension/logger.ts

```diff
@@ -0,0 +1,47 @@
+import util from 'node:util'
+
+import { window } from 'vscode'
+
+const outputChannel = window.createOutputChannel('Runme')
+
+const COLOR_REGEXP = /[\u001b\u009b][[()#;?]*(?:[0-9]{1,4}(?:;[0-9]{0,4})*)?[0-9A-ORZcf-nqry=><]/g
```

Comment:
We already have a `stripAnsi` function in `src/utils/ansi`

Reply:
replaced 👍